src/components/markdownTemplate.js

```diff
@@ -99,7 +100,11 @@ export default ({ children }) => {
 
   return (
     <main>
+      <Box paddingTop={5} paddingLeft ={2} paddingRight = {10}>
```

Comment:
Can you use padding from theme?

Reply:
default setting is spacing =8, but added it in theme for readability.